applications/DEMApplication/tests/test_restart.py

```diff
@@ -0,0 +1,84 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+from KratosMultiphysics.DEMApplication import DEM_analysis_stage
+import os
+
+class controlledExecutionScope:
```

Comment:
please use the one from [`KratosUnittest`](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/KratosUnittest.py#L178-L207)

Reply:
done